src/sonic-yang-models/yang-models/sonic-logger.yang

```diff
@@ -0,0 +1,77 @@
+module sonic-logger{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-logger";
+    prefix logger;
+
+    import sonic-types {
+        prefix stypes;
+    }
+
+    description "Logger Table yang Module for SONiC";
+    
+    typedef swss_loglevel {
+        type enumeration {
+            enum EMERG;
+            enum ALERT;
+            enum CRIT;
+            enum ERROR;
+            enum WARN;
+            enum NOTICE;
+            enum INFO;
+            enum DEBUG;
+        }
+    }
+    
+    typedef sai_loglevel {
+        type enumeration {
+            enum SAI_LOG_LEVEL_CRITICAL;
+            enum SAI_LOG_LEVEL_ERROR;
+            enum SAI_LOG_LEVEL_WARN;
+            enum SAI_LOG_LEVEL_NOTICE;
+            enum SAI_LOG_LEVEL_INFO;
+            enum SAI_LOG_LEVEL_DEBUG;
+        }
+    }
+    
+    container sonic-logger {
+
+        container LOGGER {
+
+            description "Logger table in config_db.json";
+
+            list LOGGER_LIST {
+
+                key "name";
+
+                leaf name {
+                    description "Component name in LOGGER table (example for component: orchagent, Syncd, SAI components).";
+                    type string;
+                }
+
+                leaf LOGLEVEL {
+                    description "The log verbosity for the component";
+                    mandatory true;
+                    type union {
+                        type swss_loglevel;
```

Comment:
I can still allow wrong log level to wrong component. But to resolve that we need a complex must condition, so it is up to you to add it. Thx